framework/grid/GridView.php

```diff
@@ -127,6 +127,10 @@ class GridView extends BaseListView
      * @var bool whether to show the footer section of the grid table.
      */
     public $showFooter = false;
+	/**
+	 * @var bool whether to place footer after body in DOM if $showFooter is true
```

Comment:
Please add `@since 2.0.14`.

Reply:
added below, but changes for this line still pending